UI: go to first/last page buttons in pagination

GauravBurande opened the pull request:
closes #604 
/claim #604

![image](https://github.com/Shelf-nu/shelf.nu/assets/100035026/1a25fbf6-13b5-4d4b-a81b-9ccb544faefc)

DonKoko commented on app/components/icons/library.tsx:
```diff
@@ -929,6 +929,25 @@ export const ArrowLeftIcon = (props: SVGProps<SVGSVGElement>) => (
+  >
+    <path
+      d="M15 14.1666L10.8333 9.99998L15 5.83331M9.16667 14.1666L5 9.99998L9.16667 5.83331"
+      stroke="#667085"
```
replace with `currentColor`

DonKoko commented on app/components/list/pagination/pagination.tsx:
```diff
@@ -57,6 +77,19 @@ export const Pagination = () => {
+          disabled={prevDisabled}
+          className={tw(
+            "rotate-180 rounded-none border-y-0 border-l-0 border-r border-gray-300 bg-transparent px-4 py-3 hover:bg-transparent",
+            prevDisabled && "opacity-50"
```
This should be `nextDisabled`

DonKoko reviewed (changes requested):
Okey so I added a few comments in the code and I have some general comments:
<img width="565" alt="Screenshot 2024-02-01 at 14 11 59" src="https://github.com/Shelf-nu/shelf.nu/assets/7840007/2478e4e9-afa7-461b-8bfd-d462a00c02de">

- [x] The prev/next arrows are not changed to singular chevrons(see issue, there its explained). You have to use the component <ChevronRight>
- [x] The height of those buttons dont match the height of the page selector. Make sure they fit properly

GauravBurande commented:
> Okey so I added a few comments in the code and I have some general comments: <img alt="Screenshot 2024-02-01 at 14 11 59" width="565" src="https://private-user-images.githubusercontent.com/7840007/301499735-2478e4e9-afa7-461b-8bfd-d462a00c02de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3OTYwMzIsIm5iZiI6MTcwNjc5NTczMiwicGF0aCI6Ii83ODQwMDA3LzMwMTQ5OTczNS0yNDc4ZTRlOS1hZmE3LTQ2MWItOGJmZC1kNDYyYTAwYzAyZGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDFUMTM1NTMyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGM4OWVjY2IwZGZkNDE1ZTY1NWFhNDNkYmM4YWViM2VmZmUxMjMxNGMxODZmYmY5MjkxNmVhMTllMzJmYzQzYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.xkVeERMmahat9hzCbXlNRdg0vUhMPLmEZ9cDq0kFS5A">
> 
> * [ ]  The prev/next arrows are not changed to singular chevrons(see issue, there its explained). You have to use the component
> * [ ]  The height of those buttons dont match the height of the page selector. Make sure they fit properly

fixed. The height is matched and replaced arrows with chevrons
